cola/widgets/standard.py

```diff
@@ -89,11 +96,21 @@ def export_state(self):
         """Exports data for view save/restore"""
         state = self.windowState()
         maximized = bool(state & Qt.WindowMaximized)
+        width, height = self.width(), self.height()
+
+        # when maximized we don't want to overwrite saved width/height with
+        # desktop dimensions.
+        if maximized:
+            try:
+                old_size = self._unmaximized_size
+                width, height = old_size.width(), old_size.height()
+            except:
+                width, height = None, None
```

Comment:
Wow, nice use of the `resizeEvent()` to capture the original size.  Is there any concern about `None` getting into the settings file?  It seems like that can happen since we unconditionally add `width` and `height` to the returned dictionary.  Is that ok?